sfdx-source/apex-mocks/main/classes/fflib_IDGenerator.cls

```diff
@@ -34,6 +34,13 @@ public with sharing class fflib_IDGenerator
 	public static Id generate(Schema.SObjectType sobjectType)
 	{
 		String keyPrefix = sobjectType.getDescribe().getKeyPrefix();
+		//	Deal with null keyprefix
+		//		OpportunityHistory will have keyPrefix 008 from getKeyPrefix
+		//		All other xxxHistory, xxx__History, and OpportunityFieldHistory coerce to 017 (legacy from EntityHistory)
+		if (keyPrefix == null) {	
+			keyPrefix =  sobjectType.getDescribe().getName().endsWith('History') ? '017' : null;
```

Comment:
How can we configure the 'History' and '017' strings in a manner that would not mandate a hotfix if it were to change for some wild reason or a new related need arises -- CMT record, else?

Reply:
@ImJohnMDaniel  and @stohn777  -- per our convo -- should we consider an overloaded `generate` that allows a key prefix to be supplied? Also, does it really matter what we select here? If it's ABC or 017 or anything else, it shouldn't matter

Let's consider moving the "017" up into a constant and document how we selected it